packages/client/src/i18n/messages/views/review.ts

```diff
@@ -191,7 +191,7 @@ const messagesToDefine = {
   },
   reviewActionDescriptionIncomplete: {
     defaultMessage:
-      'By sending this incomplete declaration, there will be a digital record made.\n\nTell the informant that they will receive an {deliveryMethod} with a tracking ID. They will need this to complete the declaration at a registration office within 30 days. The informant will need to provide all mandatory information before the {eventType, select, birth {birth declaration} death {death declaration}} can be registered.',
+      'Please ensure all fields are either empty or have a valid value in them.',
```

Comment:
Is this intentional? The copy change seems pretty drastic

Reply:
Yeah, this is, as discussed with @rikukissa